src/meta/rule.schema.json

```diff
@@ -89,9 +101,30 @@
             "type": "object"
         }
     },
-    "required": [
-        "given",
-        "then"
-    ],
-    "type": "object"
+            "required": [
+                "given",
+                "then"
+            ],
+            "type": "object"
+        },
+        {
+            "$ref": "http://stoplight.io/schemas/rule.schema.json#/definitions/HumanReadableSeverity"
```

Comment:
Can just be `#/definitions/HumanReadableSeverity` since it's referencing something in this file. Same for the rest of the $refs that target their own file.